ports/winflexbison/vcpkg.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "winflexbison",
+  "version-string": "2.5.24",
```

```suggestion
  "version": "2.5.24",
```
The package versions should be more proper. Please see https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#22-package-versions